cli/determined_cli/tensorboard.py

```diff
@@ -151,8 +159,10 @@ def tensorboard_config(args: Namespace) -> None:
                      "allowed per TensorBoard instance"),
             Arg("--no-browser", action="store_true",
                 help="don't open TensorBoard in a browser after startup"),
+            Arg("--hook", default=None, type=Path,
```

Comment:
blocking: This should all just be framed as handling a context directory, the same as shells and commands do. There's nothing specific to startup hooks in the code.

If you still want to do something for startup hooks specifically, then it should be possible to specify a single file to use as the hook (in which case the local name of the file doesn't have to matter).